assets/parquet/methodology.md

```
@@ -0,0 +1,90 @@
+# Parquet (PyArrow+Presto+Hive) methodology
+
+## Basics
+
+pyarrow 20.0.0
+Presto Version: [presto-server-0.293-SNAPSHOT][presto]
+Velox Version: [Velox][velox]
+
+## Setup
+
+### Json String
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)